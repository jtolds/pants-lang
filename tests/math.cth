print (3 /. 4)
print (10 /. 2)
print (- 1)
print (2 -. 1)
print (10 *. 2)
print (1 -. 2)
print (5 %. 3)
print (10 %. 5)
