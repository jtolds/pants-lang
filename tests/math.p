println (3 /. 4)
println (10 /. 2)
println (- 1)
println (2 -. 1)
println (10 *. 2)
println (1 -. 2)
println (5 %. 3)
println (10 %. 5)
