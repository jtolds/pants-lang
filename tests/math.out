0.750000
5
-1
1
20
-1
2
0

return code: 0
